gen-app-builder/data-store-status-checker/examples/data_store_checker.ipynb

```diff
@@ -0,0 +1,433 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "7be72547515d"
+      },
+      "source": [
+        "# Data Store Status Checker\n",
+        "\n",
+        "<table align=\"left\">\n",
+        "\n",
+        "  <td>\n",
+        "    <a href=\"https://colab.research.google.com/github/GoogleCloudPlatform/generative-ai/blob/main/retrieval-augmented/enterprise-search/examples/question_answering.ipynb\">\n",
+        "      <img src=\"https://cloud.google.com/ml-engine/images/colab-logo-32px.png\" alt=\"Colab logo\"> Run in Colab\n",
+        "    </a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://github.com/GoogleCloudPlatform/generative-ai/blob/main/retrieval-augmented/enterprise-search/examples/question_answering.ipynb\">\n",
+        "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
+        "      View on GitHub\n",
+        "    </a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://console.cloud.google.com/vertex-ai/workbench/deploy-notebook?download_url=https://raw.githubusercontent.com/GoogleCloudPlatform/generative-ai/blob/main/retrieval-augmented/enterprise-search/examples/question_answering.ipynb\">\n",
+        "      <img src=\"https://lh3.googleusercontent.com/UiNooY4LUgW_oTvpsNhPpQzsstV5W8F7rYgxgGBD85cWJoLmrOzhVs_ksK_vgx40SHs7jCqkTkCk=e14-rj-sc0xffffff-h130-w32\" alt=\"Vertex AI logo\">\n",
+        "      Open in Vertex AI Workbench\n",
+        "    </a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "11a41975be67"
+      },
+      "source": [
+        "---\n",
+        "\n",
+        "\n",
+        "* Author: Patrick Marlow\n",
+        "* Created: 07/17/2023\n",
+        "\n",
+        "---\n",
+        "\n",
+        "## Objective\n",
+        "\n",
+        "Simple notebook that uses the Cloud DiscoveryEngine API to check a Data Store for indexed docs.\n",
+        "\n",
+        "---\n",
+        "\n",
+        "This notebook utilizes the [google-cloud-discoveryengine](https://cloud.google.com/python/docs/reference/discoveryengine/latest) Python library.  \n",
+        "This notebook allows the user to perform the following tasks:\n",
+        "\n",
+        "- ✅ Check Indexing Status of given Data Store ID.\n",
+        "- ✅ List all documents in a given Data Store ID.\n",
+        "- ✅ List all indexed URLs for a given Data Store ID\n",
+        "- ✅ Search all indexed URLs for a specific URL within a given Data Store ID.\n",
+        "\n",
+        "---\n",
+        "\n",
+        "**References:**\n",
+        "\n",
+        "- [Google Cloud DiscoveryEngine API](https://cloud.google.com/python/docs/reference/discoveryengine/latest)\n"
```

Comment:
Should the API docs version be pinned in this docs link to match the version imported in the notebook?

Reply:
ack, pinned version 0.9.1 (currently latest)